Symfony/CS/ConfigInterface.php

```diff
@@ -59,6 +59,8 @@ public function getFixers();
      * Sets the root directory of the project.
      *
      * @param string $dir The project root directory
+     *
+     * @return Config The same instance
```

Comment:
For me it makes sense to show that the method (should) return the same instance.